customization-base/src/main/java/com/azure/autorest/customization/implementation/ls/models/SymbolKind.java

```diff
@@ -31,26 +34,32 @@
     OPERATOR(25),
     TYPE_PARAMETER(26);
 
-    private int value;
+    private static final Map<Integer, SymbolKind> INT_TO_KIND_MAP;
+
+    static {
+        INT_TO_KIND_MAP = new HashMap<>();
+
+        for (SymbolKind kind : SymbolKind.values()) {
+            INT_TO_KIND_MAP.putIfAbsent(kind.value, kind);
```

Comment:
Why use `putIfAbsent` instead of just `put`? If the value of `SymbolKind` enum is a duplicate, should that be thrown as an error?

Reply:
That's a good point, though I don't expect it to occur since the enum is fairly static